static/src/javascripts/es6/projects/common/modules/crosswords/main.js

```diff
@@ -347,14 +347,25 @@ class Crossword extends React.Component {
 
     focusClue (x, y, direction) {
         const clues = helpers.cluesFor(this.clueMap, x, y);
+        const clue = clues[direction];
 
-        if (clues && clues[direction]) {
+        if (clues && clue) {
             this.focusHiddenInput(x, y);
 
             this.setState({
                 cellInFocus: { x: x, y: y },
                 directionOfEntry: direction
             });
+
+            // Side effect
+            window.location.hash = clue.id;
```

Comment:
The list of clues now link to the hash fragment, but we also need to update the URL when a cell is selected in the grid.